### DISCUSSION TITLE
Lightning Wizard perk's 80% damage reduction does not apply to projectiles and grenades

### DISCUSSION THREAD

**Discussion by Kokhirus** [FakeFishGames/Barotrauma#14095] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I tried out to tank railgun shells with the leveled up engineer and a mechanic and got the same results despite having "Lightning Wizard" perk active.

### Reproduction steps

1. Get "Lightning Wizard" perk on an engineer.
2. Equip the engineer with the exosuit.
3. Shoot at him from railgun or throw a grenade at him.
4. Compare results with the crewmate without the "Lightning Wizard" talent.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Thanks for your report. You're correct that the description is misleading, however it is intended to only work on melee attacks (which is thematically more sensical, in our opinion), so we'll update the description.

**Comment by Kokhirus** [comment]
> ... it is intended to only work on melee attacks (which is thematically more sensical, in our opinion), so we'll update the description.

Then if that's the case there are no complaints. Thank you for your answer regarding the issue.
